include/vinecopulib/vinecop/rvine_structure.hpp

```diff
@@ -11,6 +11,56 @@
 
 namespace vinecopulib {
 
+//! R-vine structures
```

Comment:
Should we mention how it is stored (i.e., as an order vector and the upper-triangle in natural order)?

Reply:
I mention this at the end of the class docs.

Reply:
OK my bad didn't read properly